WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -151,6 +174,27 @@ class PageListViewModel @Inject constructor() : ViewModel() {
         }
     }
 
+    private fun getFeaturedImageUrl(featuredImageId: Long): String? {
```

Comment:
This might be outside of the scope of this PR, but do you think we could create a helper for this and use it in both `PostListViewModel` and here? It might be a little complicated to create that helper, because we wouldn't want to pass everything to it, like `dispatcher`, `site` etc and maybe want to pass in reusable(?) closures instead. I don't even know if that makes sense without an example, so let me know if you want me to try it out and share an approach.

Reply:
I tried to come up with something but it turned out to be pretty ugly/complicated. Let's leave it for to another PR.